kubernetes/leapp-scratch-image/6/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM scratch
+
+ADD leapp6.tar /
+
+CMD ["/sbin/init"]
```

Comment:
I was able to create a Centos 6 container (in OpenShift) using the image `gazdown/leapp-scratch:7`. Would it make sense to have just one image?